packages/elastic-apm-synthtrace/src/lib/stream_processor.ts

```diff
@@ -204,6 +219,13 @@ export class StreamProcessor<TFields extends Fields = ApmFields> {
         }
       }
     }
+    for (const aggregator of this.streamAggregators) {
```

Comment:
if this function is called for every document I can imagine that is expensive? should we memoize it by processor.event?

Reply:
Possibly, I'd rather implement a few more stream aggregators before attempting to memoize.